app/invocation/invocation_suggestion_card.tsx

```diff
@@ -156,6 +160,85 @@ const matchers: SuggestionMatcher[] = [
       ),
     };
   },
+  // Show warning if VM is low on disk/memory.
+  ({ model, runnerExecution, buildLogs }) => {
+    if (!runnerExecution) return null;
+    // TODO: show for remote bazel as well one we have a way to configure the
+    // runner via buildbuddy.yaml?
+    if (!model.isWorkflowInvocation()) return null;
+
+    let rootDiskUsage = 0.0;
+    let rootDiskBytesUsed = 0;
+    let rootDiskBytesTotal = 0;
+    // Get execution metadata from the cached ExecutionResponse,
+    // since we don't store this in the DB currently.
+    const executionMetadata = runnerExecution?.executeResponse?.result?.executionMetadata;
+    if (!executionMetadata) return null;
+
+    for (const fs of executionMetadata.usageStats?.peakFileSystemUsage ?? []) {
+      if (fs.target === "/" && Number(fs.totalBytes) !== 0) {
+        rootDiskUsage = Number(fs.usedBytes) / Number(fs.totalBytes);
+        rootDiskBytesUsed = Number(fs.usedBytes);
+        rootDiskBytesTotal = Number(fs.totalBytes);
+      }
+    }
+    const memoryUsedBytes = Number(executionMetadata?.usageStats?.peakMemoryBytes);
+    const memoryTotalBytes = Number(executionMetadata?.estimatedTaskSize?.estimatedMemoryBytes);
+    let memoryUsage = 0.0;
+    if (memoryTotalBytes !== 0) {
+      memoryUsage = memoryUsedBytes / memoryTotalBytes;
+    }
+
+    const lowResources: string[] = [];
+    const yamlSuggestions: string[] = [];
+    if (rootDiskUsage > 0.98) {
+      lowResources.push(`disk (${formatBytes(rootDiskBytesUsed)} used of ${formatBytes(rootDiskBytesTotal)} total)`);
+      // Conservatively suggest adding 5GB for now.
```

Comment:
Eh, I don't know what a "good" default here is, but this nearly triples the default, right?

Reply:
workflows default to 20GB of disk

Reply:
Oh, sorry in the screenshot I was explicitly requesting 2GB of disk in buildbuddy.yaml, overriding the default, if that's what you're referring to

Reply:
i thiiiink I just was remembering the 2GB workspace dir slush size and combined that with your screenshot to think i knew something :P